coderd/database/migrations/000292_generate_default_preset_parameter_ids.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE template_version_preset_parameters
+ALTER COLUMN id SET DEFAULT gen_random_uuid();
```

Comment:
So we're going with default generated UUIDs now? If so i'll adjust my migrations.

Reply:
I don't think a simple pattern as to whether to use or not use a function should override professional judgement.
gen_random_uuid() is appropriate in this one case as evidenced by a test that broke when I took it out. I'm putting it back here because it makes the most sense. notice I haven't put it back for the template_version_presets table.

Reply:
No argument from me, I certainly prefer it, but then we should be consistent with its use.